src/Symfony/Component/DomCrawler/Crawler.php

```diff
@@ -554,13 +554,13 @@ public function nodeName()
     /**
      * Returns the node value of the first node of the list.
      *
-     * @param mixed $default When provided and the current node is empty, this value is returned and no exception is thrown
+     * @param string|null $default The value to return when the current node is empty
      *
-     * @return string The node value
+     * @return string|null The node value
```

Comment:
should we actually allow returning `null`? I feel like we could make `null` mean "throw"

Reply:
updated